setup.py

```diff
@@ -59,6 +59,12 @@
                 'adminapi=adminapi.__main__:main',
             ],
         },
+        extras_require={
```

Comment:
Good catch to add these. But shouldn't this be done via `install_requires` since adminapi does not work without them and `extas_requires` lists optional requirements ?

> A dictionary mapping names of “extras” **(optional features of your project)** to strings or lists of strings specifying what other distributions must be installed to support those features. See the section on Declaring required dependency for details and examples of the format of this argument.

Source: https://setuptools.readthedocs.io/en/latest/references/keywords.html (extras_require)

Reply:
The confusion here is different packages at one setup. The requirements are for `adminapi`, but here is `serveradmin` itself too. So, I don't know how to handle it propely

Reply:
Ah yes. I remember the problem - true.

However in this case I think it is fine because in the backend (Serveradmin) we need both paramiko and netaddr:
- https://github.com/innogames/serveradmin/blob/master/serveradmin/apps/models.py#L10
- https://github.com/innogames/serveradmin/blob/master/serveradmin/serverdb/models.py#L12

So I suggest we put it to `install_requires`. Does that make sense to you ?

Reply:
Sounds perfect :+1: